Scripts/RemediationScripts/ControlRemediationSpecification.json

```diff
@@ -122,17 +122,36 @@
                     "Arguments":["SubscriptionId"]
                 }
         },
-        {
-            "ControlId": "Azure_CloudService_SI_Disable_RemoteDesktop_Access",
-            "EnableRemediation": "true",
-            "RemediationScriptUrl": "https://raw.githubusercontent.com/azsk/AzTS-docs/main/Scripts/RemediationScripts/Remediate-RemoteDesktopAccess.ps1",
-            "LoadCommand": "Remediate-RemoteDesktopAccess.ps1",     
-            "InitCommand": "Disable-RemoteDesktopAccess",
-            "RollbackMetadata":
-                {
-                    "RollbackCommand": "",
-                    "Arguments":["SubscriptionId"]
-                }
+      {
+        "ControlId": "Azure_CloudService_SI_Disable_RemoteDesktop_Access",
+        "EnableRemediation": "true",
+        "RemediationScriptUrl": "https://raw.githubusercontent.com/azsk/AzTS-docs/main/Scripts/RemediationScripts/Remediate-RemoteDesktopAccess.ps1",
+        "LoadCommand": "Remediate-RemoteDesktopAccess.ps1",
+        "InitCommand": "Disable-RemoteDesktopAccess",
+        "RollbackMetadata": {
+          "RollbackCommand": "",
+          "Arguments": [ "SubscriptionId" ]
         }
+      },
+      {
+      "ControlId": "Azure_ServiceFabric_DP_Set_Property_ClusterProtectionLevel",
+      "EnableRemediation": "true",
+      "RemediationScriptUrl": "https://raw.githubusercontent.com/azsk/AzTS-docs/main/Scripts/RemediationScripts/Remediate-SetClusterProtectionLevelForServiceFabric.ps1",
+      "LoadCommand": "Remediate-SetClusterProtectionLevelForServiceFabric.ps1",
+      "InitCommand": "Set-ClusterProtectionLeveltoEncryptandSignforServiceFabric",
+        "RollbackMetadata": {
+          "RollbackCommand": "Set-ClusterProtectionLeveltoPreviousValueforServiceFabric",
+          "Arguments": [ "SubscriptionId", "RollBackType" ]
+        },
+        {
+          "ControlId": "Azure_ServiceFabric_DP_Set_Property_ClusterProtectionLevel",
```

Comment:
Why two cluster protection control object, also, revert, `Azure_CloudService_SI_Disable_RemoteDesktop_Access` to previous state, only changes visible in this file should be your added object.